File: Android/app/src/main/java/com/accelo/data/model/Meta.kt
package com.accelo.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName


/**
 * Created by Yuri Popiv on 11/25/2019.
 */
data class Meta(
    @SerializedName("message")
    @Expose
    var message: String? = null,
    @SerializedName("more_info")
    @Expose
    var moreInfo: String? = null,
    @SerializedName("status")
    @Expose
    var status: String? = null
)

File: iOS/Accelo/Model/Meta.m
//
//  Meta.m
//  Accelo
//
//  Created by Kuryliak Maksym on 06.12.2019.
//  Copyright © 2019 Wise-Engineering. All rights reserved.
//

#import "Meta.h"

@implementation Meta

+ (NSDictionary *)JSONKeyPathsByPropertyKey {
    return @{
        @"message": @"message",
        @"moreInfo": @"more_info",
        @"status": @"status"
    };
}

@end
